projects/angular2-jsonapi/src/models/json-api.model.spec.ts

```diff
@@ -37,17 +34,17 @@ describe('JsonApiModel', () => {
           name: 'Daniele',
           surname: 'Ghidoli',
           date_of_birth: '1987-05-25',
-          school: { name: 'Massachusetts Institute of Technology', students: 11319, foundation: '1861-10-04' }
+          school: {name: 'Massachusetts Institute of Technology', students: 11319, foundation: '1861-10-04'}
```

Comment:
Spacing around object declarations isn't supported in TSLint: https://github.com/palantir/tslint/issues/1044

Eventually, Angular CLI will switch to ESLint: https://github.com/angular/angular-cli/issues/13732